src/app/components/articles/article-type1/article-type1.component.scss

```diff
@@ -0,0 +1,34 @@
+@use "/src/styles.scss" as *;
+
+.article-type1 {
+    //position:relative;
+    height:700px;
+}
+
+img {
+    width:1050px;
+}
+.blur {
+position: relative;
+width: 1050px;
+height: 150px;
+left: 0px;
+bottom: 170px;
+
+backdrop-filter: blur(9.5px);
+}
+
+.description {
+    margin-left:1rem;
+}
+
+h2 {
+    color:$category-health-color;
+    font-weight: normal;
+}
+
+h4,h6 {
```

Comment:
same here
You can use Scss imbrication to avoid putting classes on every little html tag if you want